library/src/main/scala-new-collections/scala/scalajs/js/Any.scala

```diff
@@ -181,13 +201,15 @@ object Any extends LowPrioAnyImplicits {
 }
 
 trait LowPrioAnyImplicits extends LowestPrioAnyImplicits {
-  implicit def wrapArray[A](array: Array[A]): WrappedArray[A] =
-    new WrappedArray(array)
-  implicit def wrapDictionary[A](dict: Dictionary[A]): WrappedDictionary[A] =
-    new WrappedDictionary(dict)
+  implicit def wrapArray[A](array: js.Array[A]): js.WrappedArray[A] =
+    new js.WrappedArray(array)
+  implicit def wrapDictionary[A](dict: js.Dictionary[A]): js.WrappedDictionary[A] =
+    new js.WrappedDictionary(dict)
 }
 
 sealed trait LowestPrioAnyImplicits {
-  implicit def iterableOps[A](iterable: Iterable[A]): IterableOps[A] =
-    new IterableOps(iterable)
+  implicit def arrayAsIterable[A](array: js.Array[_ <: A]): scala.collection.Iterable[A] =
+    new js.WrappedArray(array)
```

Comment:
And I had to add this new implicit to take precedence over `iterableOps`, in case the expected type is already known to be `scala.collection.Iterable[A]` but the type of the array is a `js.Array[B]` with `B <: A`. This may be due to subtle changes in type inference in Scala 2.13.0-M5.